docs/general/administration/hardware-selection.md

```diff
@@ -88,89 +86,56 @@ When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will r
 
-Encoder Quality: Intel > Nvidia > AMD > Apple
-
-Intel graphics is recommended over Nvidia graphics because of the quality. AMD or Apple Silicon are not recommended since they produce poor quality H.264 and H.265(HEVC) output.
+Encoder Quality: Apple ≥ Intel > Nvidia >>> AMD
 
-:::note A Note on AMD AV1 Encoders
+Intel graphics is recommended over Nvidia graphics because of the quality. AMD is not recommended because of poor quality H.264 and H.265(HEVC) output.
 
-With AV1, AMD has significantly improved the quality of their encoders. However, Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1. Even after Jellyfin adds support for transcoding to AV1, transcoding to H.264 or H.265(HEVC) is still more likely due to AV1 being in the early stages of adoption.
-
-:::
+AMD has significantly improved AV1 encoder quality. However you are still more likely to transcode to H.264 or H.265 than to AV1 due to compatibility.
 
 A list of common codecs can be found [here](/docs/general/clients/codec-support/)
 
-The following is a list of codecs Jellyfin supports transocding to:
+The following is a list of video codecs Jellyfin supports transocding to:
 
 - H.264 (Most common transcode target)
 - H.265 (Limited supported by clients)
-
-:::note AV1 Support
-
-Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1 as a target. However, this might change in the future. Source content in AV1 will transcode to other codecs normally if needed.
-
-:::
+- AV1 (New in 10.9, supported by most modern browsers)
 
 For decoding support, the more codecs the better. However, there are only a few codecs that media is commonly available in, as listed above.
 
 #### Intel Graphics
 
 Please refer to [this table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
 
-:::caution F series CPUs
-
-Intel CPUs with a model description that ends with F don't have integrated graphics. These are intended to be used with a dedicated graphics card.
+Intel CPUs with a model description that ends with F don't have integrated graphics. These are intended to be used with a dedicated graphics card. Please beware if you intend to use Intel integrated graphics.
 
-:::
-
-:::note Intel 12th Gen, 13th Gen and ARC Support
-
-If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version. It is also recommended that `jellyfin-ffmpeg6` be installed instead of the default `jellyfin-ffmpeg5` on these hardware platforms.
-
-:::
+If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version.
```

Comment:
A link to the "known limitations" section of Intel acceleration could also be helpful as Intel breaks their drivers from time to time and will have more constraints on the kernel version.

Reply:
Linked.